src/include/ipc/topology.h

```diff
@@ -83,11 +83,22 @@ struct sof_ipc_comp {
 #define SOF_MEM_CAPS_CACHE			(1 << 6) /**< cacheable */
 #define SOF_MEM_CAPS_EXEC			(1 << 7) /**< executable */
 
+#define SOF_BUF_OVERRUN_PERMITTED	BIT(0) /**< overrun will cause ring
```

Comment:
Pls use the following dox format for longer descriptions:
```c
/**
 * Overrun will cause ....
 */
#define ....
```

Reply:
revised